### PR TITLE
feature(cb2-7330): work to retain the historic vrms when user clicks arch…

### PR DESCRIPTION
**Pull request by owen-corrigan-bjss**
CB2-7330:

Historic vrm's were not populated when clicking the archive button, this fix should sort that.

### CODE REVIEW

File: tests/integration/archiveTechRecordStatus.intTest.ts [hunk 1]
```diff
@@ -19,16 +19,15 @@ describe("archiveTechRecordStatus", () => {
   beforeAll(async () => {
     jest.restoreAllMocks();
     // await emptyDatabase();
-    await populateDatabase();
+    // await populateDatabase();
```
Comment (andrewf-bjss) [inline tests/integration/archiveTechRecordStatus.intTest.ts:22]: do we need this commented out line?

File: @Types/ITechRecord.d.ts [hunk 2]
```diff
@@ -124,6 +124,8 @@ export default interface ITechRecord {
   manufacturerDetails: ManufacturerDetails;
   authIntoService: AuthIntoService;
   lettersOfAuth: LettersOfAuth;
+  historicPrimaryVrm?: string;
+  historicSecondaryVrms?: string;
```
Comment (andrewf-bjss) [inline @Types/ITechRecord.d.ts:128]: as this is plural, could this be an array type?

Comment (owen-corrigan-bjss) [reply @Types/ITechRecord.d.ts:128]: Yes it should be you're absolutely right

File: tests/integration/archiveTechRecordStatus.intTest.ts (hunk 1 above)
Comment (owen-corrigan-bjss) [reply tests/integration/archiveTechRecordStatus.intTest.ts:22]: I think I just missed some on when I was deleting the others

**Review by tahayazici — APPROVED** [review]

**Review by mcadenas-bjss — APPROVED** [review]

**Review by HazelWilderspin — APPROVED** [review]

**Review by tahayazici — APPROVED** [review]

**Pull request merged**